### DISCUSSION TITLE
工作两年还机会润嘛？

### DISCUSSION THREAD

**Discussion by Fodesu** [The-Run-Philosophy-Organization/run#83] [Q&A]
在小厂工作两年了，本科GPA只有3.2😥有什么办法润嘛？🙏

**Comment by the-run-philosophy** [comment]
参考王探长的部分：

https://github.com/The-Run-Philosophy-Organization/run/blob/main/%E6%B6%A6%E5%AD%A6%E6%96%B9%E6%B3%95%E8%AE%BA/%E5%90%84%E5%9B%BD%E9%80%89%E6%8B%A9/%E7%BE%8E%E5%9B%BD/%E7%8E%8B%E6%8E%A2%E9%95%BF%E8%B0%88%E4%B8%BA%E4%BB%80%E4%B9%88%E6%9D%A5%E4%BA%86%E5%B0%B1%E6%98%AF%E7%BE%8E%E5%9B%BD%E4%BA%BA.md

**Comment by taobowen** [comment]
随便润，GPA不影响润，只会影响你申请学校的好坏。而且工作经验加分很多，会淡化低GPA的影响。

**Comment by h7221836** [comment]
同工作两年，感觉润出去还是得读个硕士

**Comment by stackunderflow111** [comment]
你家有几个金币，工作内容是写程序吗？英语怎么样？
你除了GPA其他啥都不说的话我只能建议你窗户跳下去直接投胎，这样比较快。
